test/runtests.jl

```diff
@@ -146,3 +147,14 @@ end
   @test all(model.meta.lvar .== 0)
   @test all(model.meta.uvar .== Inf)
 end
+
+@testset "denoing_model" begin
```

Comment:
Is it possible to use the `test_objectives` function here?

Reply:
Thanks for the comments. No we can't use test_objectives because we don't have an implementation of the NLS version

Reply:
OK. Is it possible to test the objective you implemented, without the `test_objectives` function?